core/src/luigi-config/navigation/navigation-data-init.js

```diff
@@ -221,6 +223,35 @@ async function fetchNavigationData(authData, permissionSet) {
   }
 }
 
+async function getObservabilityNodes(authData, enabledFeatures) {
+  const links =
+    enabledFeatures.OBSERVABILITY?.config.links || // take the custom config at first
+    (await getClusterParams()).config.features.OBSERVABILITY?.config.links; // use the Busola configMap as a fallback
+
+  const CATEGORY = {
+    label: 'Observability',
+    icon: 'stethoscope',
+    collapsible: true,
+  };
+  const navNodes = await Promise.all(
+    links.map(async ([label, path]) => {
```

Comment:
what if links are empty? the config map can be edited to not have this field, and then the whole app is unusable

Reply:
I thought we agree to use some kind of schema for the config... If not, then what if the `config` is empty or is not an object type or the `links` are not strings or...

Reply:
That's a good point there, I think we have a JSON schema fo our config already

Reply:
I don't think we do. If we decided to create one, I suggest to do this in a separate task.